src/components/PreviewTestingView/styles.module.css

```diff
@@ -0,0 +1,4 @@
+.view {
+  height: 100vh;
+  background: linear-gradient(90deg, #007BC7 0%, #02588D 100%);
```

Comment:
I left the same comment in Aaron's PR. But let's try to keep all the colors as shared variables in theme (probably at :root)